src/Process/Pool/Logger.php

```diff
@@ -68,15 +69,21 @@ protected function _getProcess(): ProcessInterface
     public function log($level, $message, array $context = [])
     {
         if ($this->_isEnabled() === true) {
+            $logMessage = $this->getProcessPoolLoggerMessageFactory()->create();
+
             if ($this->getLevelFilterMask()[$level] === false) {
                 if ($this->_exists(ProcessInterface::class)) {
                     $processId = (string)$this->_getProcess()->getProcessId();
+                    $fromProcessInterfaceBuilder =
```

Comment:
This variable should either be `$fromProcessInterfaceBuilderFactory`, or it should store the actual builder returned by the `create()` method

Reply:
@alexberryman this feedback still stands, The variable calls this a builder, but it's actually the builder factory

Reply:
@mucha55 The current code is just creating and building a full `$serializableProcess` object:
```php
$serializableProcess = $this->getProcessPoolLoggerMessageSerializableProcessFromProcessModelBuilderFactory()
                        ->create()
                        ->setProcessModelInterface($this->_getProcess())
                        ->build();
                    $logMessage->setKojoProcess($serializableProcess);
```

Reply:
Yeah, but you're not pointing to a `Builder` you `create()`d, you're pointing to the `Builder\Factory`

Reply:
The problem was inside me all along